src/codecatalyst/vue/create/backend.ts

```diff
@@ -64,13 +67,8 @@ export class CodeCatalystCreateWebview extends VueWebview {
     ) {
         super()
 
-        // When webview first loads, an instance of this class
-        // is created.
-        // We build the prompter here since it immeditely starts
-        // fetching the Projects upon creation.
-        // When a user triggers the prompt to select a Project the **first** time,
-        // the fetching of Projects will already be in progress.
-        this.projectPrompter = createProjectPrompter(this.client)
+        // triggers pre-loading of Spaces
+        this.spacePrompter = createOrgPrompter(client)
```

Comment:
Creating a prompter, for Space or Project, immediately triggers the loading of the Resource.

So as you will see in this PR, theres some additional complexity of when a Prompter gets created.